packages/compass-data-modeling/src/components/diagram-editor.tsx

```diff
@@ -255,6 +274,21 @@ const DiagramContent: React.FunctionComponent<{
       data-testid="diagram-editor-container"
     >
       <div className={modelPreviewStyles} data-testid="model-preview">
+        {showDataInfoBanner && (
+          <Banner
+            variant="info"
+            dismissible
+            onClose={() => setshowDataInfoBanner(false)}
+            className={dataInfoBannerStyles}
+            data-testid="data-info-banner"
+          >
+            <h4>Worried about your data?</h4>
```

Comment:
Nit
- Based on Julia's comment [here], can we shorten the second sentence (https://www.figma.com/design/w51HGIb2PDdyZqyCOMMypr?node-id=1528-22764#1404509115) "Changes made to the diagram will not impact your data"
- "worried" is making me worried :) How do you feel about rephrasing as "Questions about your data?"